src/styles/blocks/image.css

```diff
@@ -0,0 +1,12 @@
+.image {
+  display: inline-block;
+}
+
+.photo {
+  width: 160px;
+  height: 134px;
+}
+
+.fullCard__image {
+  margin: 32px 19px 36px;
+}
```

Comment:
Let's follow the proper naming convention

Reply:
No sure that i get that comment right:
That is class that styling margin inside _image_  block. Because i can't style margins and position in Block class i should add extra element with styling margin. That is what i did.

Reply:
then put this stylling in `fullcard.css`

Reply:
Now understand, thank you. Missed that part